graphql-server/src/schemas/neo4j.ts

```diff
@@ -21,11 +24,13 @@ import { connect, headers, StringCodec } from "nats";
 const fetch = require("node-fetch");
 const request = require("request-promise-native");
 import { ApolloClient, ApolloQueryResult } from "apollo-client";
+import { gql } from "apollo-server";
 
 export async function buildNeo4jDbSchema() {
   const driver = neo4j.driver(
     props.neo4j_graphql_uri,
-    neo4j.auth.basic(props.neo4j_username, props.neo4j_password)
+    neo4j.auth.basic(props.neo4j_username, props.neo4j_password),
+    { disableLosslessIntegers: true } // maps Cypher Integer to JavaScript Number
```

Comment:
This allowed us to use Cypher type `Int` when extending the type definitions below. 

More context directly from the docs:
> [Integer](https://neo4j.com/docs/api/javascript-driver/current/class/lib6/integer.js~Integer.html) is not one of JavaScript’s native types, but rather a custom one accomodating Cypher’s precision. You can disable this through the [disableLosslessIntegers configuration entry](https://neo4j.com/docs/api/javascript-driver/current/function/index.html#configuration) when instantiating the driver, so that JavaScript’s native Number type is used instead. Note that this can lead to a loss of precision.

(I think loss of precision is fine as we only use integers for our application.)